api4/openGraph.go

```diff
@@ -7,12 +7,12 @@ import (
 	"net/http"
 
 	"github.com/mattermost/mattermost-server/v5/model"
-	"github.com/mattermost/mattermost-server/v5/utils"
+	"github.com/mattermost/mattermost-server/v5/services/cache/lru"
 )
 
 const OPEN_GRAPH_METADATA_CACHE_SIZE = 10000
 
-var openGraphDataCache = utils.NewLru(OPEN_GRAPH_METADATA_CACHE_SIZE)
+var openGraphDataCache = lru.NewLru(OPEN_GRAPH_METADATA_CACHE_SIZE)
```

Comment:
`lru.NewLru` causes stutter. (https://golang.org/doc/effective_go.html#package-names). It should be `lru.New`.

Reply:
Right! I think it is fixed now